Av1an/encoders/aom.py

```diff
@@ -15,22 +16,36 @@ def __init__(self):
             output_extension='ivf'
         )
 
-    def compose_1_pass(self, a: Args, c: Chunk) -> MPCommands:
+    def compose_1_pass(self, a: Args, c: Chunk, output=None) -> MPCommands:
+        if not output:
+            output = c.output
```

Comment:
Instead of putting this in every compose method, we should make output required and just pass c.output when needed.